petab/v2/C.py

```diff
@@ -129,30 +129,23 @@
 CONDITION_NAME = "conditionName"
 #: Column in the condition table with the ID of an entity that is changed
 TARGET_ID = "targetId"
-#: Column in the condition table with the type of value that is changed
-VALUE_TYPE = "valueType"
+#: Column in the condition table with the operation type
+OPERATION_TYPE = "operationType"
 #: Column in the condition table with the new value of the target entity
 TARGET_VALUE = "targetValue"
-# value types:
-VT_CONSTANT = "constant"
-VT_INITIAL = "initial"
-VT_RATE = "rate"
-VT_ASSIGNMENT = "assignment"
-VT_RELATIVE_RATE = "relativeRate"
-VT_RELATIVE_ASSIGNMENT = "relativeAssignment"
-VALUE_TYPES = [
-    VT_CONSTANT,
-    VT_INITIAL,
-    VT_RATE,
-    VT_ASSIGNMENT,
-    VT_RELATIVE_RATE,
-    VT_RELATIVE_ASSIGNMENT,
+# opeartion types:
+OT_CUR_VAL = "setCurrentValue"
+OT_NO_CHANGE = "noChange"
```

Comment:
Other constants in C.py are not shortened, so could stick to that "convention". To reduce the length, `OT`/`OPERATION_TYPE` could be dropped from the start of the name, since I don't think we have that style elsewhere either.

Reply:
I would prefer introducing prefixes elsewhere than dropping them. I find that this makes it clearer where they belong.

Reply:
Also fine to completely replace them by enums.

Reply:
I'll leave it for now, as there is a good chance that "operation types" will be dropped completely